controllers/actions.summerwind.net/suite_test.go

```diff
@@ -49,12 +48,10 @@ func TestAPIs(t *testing.T) {
 
 	config.GinkgoConfig.FocusStrings = append(config.GinkgoConfig.FocusStrings, os.Getenv("GINKGO_FOCUS"))
 
-	RunSpecsWithDefaultAndCustomReporters(t,
-		"Controller Suite",
-		[]Reporter{printer.NewlineReporter{}})
```

Comment:
The whole `printer` package has been removed in controller-runtime v0.14.0 due to the Ginkgo v2 upgrade.
See https://github.com/kubernetes-sigs/controller-runtime/pull/1977 for more context.